tools/api-markdown-documenter/src/rendering/Rendering.ts

```diff
@@ -42,9 +42,7 @@ export function renderModelDocument(
     apiModel: ApiModel,
     config: Required<MarkdownDocumenterConfiguration>,
 ): MarkdownDocument {
-    if (config.verbose) {
-        console.log(`Rendering API Model document...`);
-    }
+    config.logger.verbose(`Rendering API Model document...`);
```

Comment:
nit: consider creating a local variable with the logger to make the code a little easier to read.